<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>987</width>
    <height>722</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="font">
    <font>
     <family>Microsoft JhengHei UI</family>
    </font>
   </property>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>0</y>
      <width>211</width>
      <height>641</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft JhengHei UI</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Arm Control Panel</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
    <widget class="QGroupBox" name="groupBox1">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>180</width>
       <height>91</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Microsoft JhengHei UI</family>
       <pointsize>12</pointsize>
       <weight>50</weight>
       <bold>false</bold>
       <underline>false</underline>
      </font>
     </property>
     <property name="title">
      <string>Joint 1</string>
     </property>
     <widget class="QPushButton" name="pushButtonCCW1">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Microsoft JhengHei Light</family>
        <pointsize>12</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>CCW</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonCW1">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>30</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Microsoft JhengHei Light</family>
        <pointsize>12</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>CW</string>
      </property>
     </widget>
     <widget class="QSlider" name="horizontalSlider">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>160</width>
        <height>20</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>130</y>
       <width>180</width>
       <height>91</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Microsoft JhengHei UI</family>
       <pointsize>12</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="title">
      <string>Joint 2</string>
     </property>
     <widget class="QPushButton" name="pushButtonCCW2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Microsoft JhengHei Light</family>
        <pointsize>12</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>CCW</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonCW2">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>30</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Microsoft JhengHei Light</family>
        <pointsize>12</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>CW</string>
      </property>
     </widget>
     <widget class="QSlider" name="horizontalSlider_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>160</width>
        <height>20</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>240</y>
       <width>180</width>
       <height>91</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Microsoft JhengHei UI</family>
       <pointsize>12</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="title">
      <string>Joint 3</string>
     </property>
     <widget class="QPushButton" name="pushButtonCCW3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Microsoft JhengHei Light</family>
        <pointsize>12</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>CCW</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonCW3">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>30</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Microsoft JhengHei Light</family>
        <pointsize>12</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>CW</string>
      </property>
     </widget>
     <widget class="QSlider" name="horizontalSlider_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>160</width>
        <height>20</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>350</y>
       <width>191</width>
       <height>141</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Microsoft JhengHei UI</family>
       <pointsize>12</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="title">
      <string>Paint Brush</string>
     </property>
     <widget class="QPushButton" name="pushButtonPaint">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>40</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Microsoft JhengHei Light</family>
        <pointsize>12</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Paint</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonUndo">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Microsoft JhengHei Light</family>
        <pointsize>12</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Undo</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonClear">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>80</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Microsoft YaHei Light</family>
        <pointsize>12</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Clear</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>110</y>
        <width>181</width>
        <height>17</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Microsoft JhengHei Light</family>
        <pointsize>12</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Paint Continuously</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>761</width>
      <height>651</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>987</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
